rough/src/svg2roughjs.js

```
@@ -0,0 +1,86 @@
+  if (!content) {
+    throw new Error('Could not save file, no sketch given.')
+  }
+  // TODO validate SVG?
+
+  fs.writeFileSync(outputFilePath ?? './sketch.svg', content)
+}
+
+function getSvg2RoughjsArgs(argv) {
+  const args = { ...argv }
+  // remove arguments for the CLI
+  delete args._
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)